src/main/java/edu/nextstep/racingcar/car/Cars.java

```diff
@@ -0,0 +1,29 @@
+package edu.nextstep.racingcar.car;
+
+import edu.nextstep.racingcar.strategy.RandomStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private List<RacingCar> cars = new ArrayList<>();
+
+    public Cars(List<RacingCar> racingCarList) {
+        this.cars = racingCarList;
+    }
+
+    public void playRacingOneRound() {
+        for (RacingCar car : cars) {
+            car.move(new RandomStrategy());
+        }
+    }
```

Comment:
```suggestion
    public void playRacingOneRound(RacingStrategy racingStrategy) {
        for (RacingCar car : cars) {
            car.move(racingStrategy);
        }
    }
```
이렇게 메소드를 실행할 때 주입해주면 어떨까요? 지금은 RandomStrategy로 고정되어서 유연하지 않은 메소드랍니다! 무엇보다 `playRacingOneRound()`에 대한 정확한 테스트가 불가능한 상태랍니다.